telemetry/definitions/commonDefinitions.json

```diff
@@ -1549,6 +1549,11 @@
             "type": "string",
             "description": "Metric-defined identifier"
         },
+        {
+            "name": "id2",
+            "type": "string",
+            "description": "x-amz-id-2 for S3 Upload Debugging"
```

Comment:
Its great to see an example, but please add a proper description to explain the field.